### ISSUE TITLE
Calling layout on mCircleView twice

### ISSUE THREAD

**Issue by agentraymond232** [ashakirov/Horizontal-SwipeRefreshLayout#1]
https://github.com/ashakirov/Horizontal-SwipeRefreshLayout/blob/d06dba9407b6e75b21de9367e83e8eafaa296369/Example/HSwipeRefreshLayout/src/main/java/custom/widget/SwipeRefreshLayout.java#L578

Are you sure you need to call layout method on mCircleView twice? I was comparing your code with the Official SwipeRefreshLayout and saw that they had called the layout method on mCircleView just once. It also does not make any sense to call it twice on the same view, thus overriding the effect of the first method call.

Did you intent to use mProgress here instead of mCircleView?

**Comment by ashakirov** [comment]
Yes, it looks like I wrote horizontal layouting and just forgot to remove original one. Thanks!
